cachito/workers/config.py

```diff
@@ -64,6 +64,10 @@ class Config(object):
     cachito_task_log_format = (
         "[%(asctime)s #%(request_id)s %(name)s %(levelname)s %(module)s.%(funcName)s] %(message)s"
     )
+    # How many times a worker tries to check the packages file before finalizing a request
+    check_packages_file_max_retries = 10
+    # Interval between checks for the packages file, in seconds
+    check_packages_file_interval = 1.0
```

Comment:
Cachito-specific options should have the `cachito_` prefix and should be documented in the README